internal/version/check_update.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package version
+
+import (
+	"context"
+
+	"github.com/Masterminds/semver"
+
+	"github.com/elastic/elastic-package/internal/github"
+	"github.com/elastic/elastic-package/internal/logger"
+)
+
+const (
+	repositoryOwner = "elastic"
+	repositoryName  = "elastic-package"
+)
+
+// CheckUpdate function checks using Github Release API if newer version is available.
+func CheckUpdate() {
+	if Tag == "" {
+		logger.Debugf("Distribution built without a version tag, can't determine release chronology. Please consider using official releases at " +
+			"https://github.com/elastic/elastic-package/releases")
+		return
+	}
+
+	githubClient := github.UnauthorizedClient()
+	release, _, err := githubClient.Repositories.GetLatestRelease(context.Background(), repositoryOwner, repositoryName)
```

Comment:
Nit. Use `context.TODO()` in case at some moment we pass a context here?
```suggestion
	release, _, err := githubClient.Repositories.GetLatestRelease(context.TODO(), repositoryOwner, repositoryName)
```

Reply:
Good idea, committed, thanks!